client/js/ui.js

```diff
@@ -208,11 +209,20 @@ UI.prototype = {
   },
 
   makeBox: function () {
-    var boxTemplate = document.getElementById('box-template').innerHTML;
-    var box = document.createElement('div');
-    box.innerHTML = boxTemplate;
+      var boxTemplate = document.getElementById('box-template').innerHTML;
+      var box = document.createElement('div');
+      var name = this.makeBackground();
+      var url = "url('" + name +  "')"
+      box.innerHTML = boxTemplate;
+      box.getElementsByTagName('div')[0].style["background-image"] = url;
+      return box.getElementsByTagName('div')[0];
+  },
 
-    return box.getElementsByTagName('div')[0];
+  makeBackground: function() {
+      if(this.backgrounds && this.backgrounds.length) {
+	  var bgix = Math.floor( this.backgrounds.length * Math.random( ));
```

Comment:
Would be cool if this was somehow identical for all participants of the chat, but I never did expect a PR for an 8-year old weekend project 👍